src/convertToObject.js

```diff
@@ -6,7 +6,14 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  return sourceString.split(';').reduce((finalStyle, part) => {
+    if (part.trim()) {
+      const [command, value] = part.split(':');
+
+      finalStyle[command.trim()] = value.trim();
+    }
 
+    return finalStyle;
```

Comment:
[CODE STYLE] - When chaining methods, such as `split` and `reduce`, ensure that each method call starts on a new line with the same indentation to improve readability. Good example:

const result = sourceString
  .split(';')
  .reduce((finalStyle, part) => { /*...*/ }, {});